src/network/networkd-manager.c

```diff
@@ -38,7 +38,7 @@ static int setup_default_address_pool(Manager *m) {
 
         /* Add in the well-known private address ranges. */
 
-        r = address_pool_new_from_string(m, &p, AF_INET6, "fc00::", 7);
+        r = address_pool_new_from_string(m, &p, AF_INET6, "fd00::", 8);
```

Comment:
so what does this mean precisely? that only half the previous range is now among the default? is that correct?

Reply:
`fc00::/8` may be used in the future. From [rfc4193](https://tools.ietf.org/html/rfc4193#section-3.1), about the 8th bit,
> Set to 1 if the prefix is locally assigned.
> Set to 0 may be defined in the future.